courses/free-courses-en.md

```diff
@@ -405,6 +406,7 @@
 * [C++ Programming Course - Beginner to Advanced](https://www.youtube.com/watch?v=8jLOx1hD3_o) - Daniel Gakwaya (freeCodeCamp)
 * [C++ Programming Language - C++ Tutorial](https://www.youtube.com/playlist?list=PLVlQHNRLflP8_DGKcMoRw-TYJJALgGu4J) - Naresh i Technologies
 * [C++ Programming Tutorial for Beginners (For Absolute Beginners)](https://www.youtube.com/playlist?list=PLS1QulWo1RIYSyC6w2-rDssprPrEsgtVK) - ProgrammingKnowledge
+* [C++ STL Complete Tutotial in One Video](https://youtu.be/RRVYpIET_RU?si=cGarnTa6IoVyTRby) - Striver (take U forward)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes